test/DebuggerTesting/OpenDebug/Events/StoppedEvent.cs

```diff
@@ -63,6 +64,18 @@ public class StoppedEvent : Event<StoppedEventValue>
         private bool verifyLineRange;
         private int startLine;
         private int endLine;
+        private ulong address;
+
+        public StoppedEvent(ulong address)
+            : base("stopped")
+        {
+
+            this.address = address;
```

Comment:
Since we dont really send the MemoryReference or set the source correctly, is there a good way to validate the address we stopped at?

Reply:
@gregg-miskelly Thoughts on how to properly test hitting the instruction bp address?

Reply:
Do you mean we aren't adding the memory reference to the stack frame? If so, can we just fix that?

Reply:
There is no address on the StoppedEvent.

I was wondering if there was a way to use the `address` in `StoppedEvent` for `ProcessActualResponse`.
https://github.com/microsoft/MIEngine/blob/829f18a4c837dec48e0b51a644c9f42ff001454e/test/DebuggerTesting/OpenDebug/Events/StoppedEvent.cs#L153

I can add additional calls in the test afterwards that queries the stack information and asserts the first frame's memoryReference is the Instruction BP's address.

Reply:
I would suggest adding the extra calls to ask for the stack trace. Then we aren't verifying something that is a fake scenario anyway.